server/helpers/dates.js

```diff
@@ -0,0 +1,5 @@
+import format from 'date-fns/format';
+
+export const TODAY_WORDS = format(new Date(), 'd MMM yyyy');
```

Comment:
TODAY_WORDS doesn't make any sense to someone reading the code for the first time as it has multiple meanings.

Also, the date will change everyday so it is a bad idea to make it a constant. A function is more preferable.

```suggestion
export const getTodaysDateInWord = () => format(new Date(), 'd MMM yyyy');
```